lib/routes/foodtalks/index.ts

```diff
@@ -7,7 +7,8 @@ import logger from '@/utils/logger';
 export const route: Route = {
     path: '/',
     categories: namespace.categories,
-    example: '/foodtalks',
+    example: '/foodtalks?limit=30',
+    parameters: { limit: 'number of articles being fetched' },
```

Comment:
```suggestion
    example: '/foodtalks',
```
No need to mention the common parameter in example